third_party/cpp-base64/base64.cc

```diff
@@ -0,0 +1,285 @@
+/*
+   base64.cpp and base64.h
+
```

Comment:
Why not just use git_repository with https://github.com/ReneNyffenegger/cpp-base64?